requirements.txt

```diff
@@ -191,11 +191,12 @@ rsa==4.0
 s3transfer==0.3.3
```

Comment:
Note your `requirements.txt` should contain the *minimal* libraries your *users* need to have installed to run your app, and that you need to regenerate data and models. It shouldn't contain any libraries that you used for experiments but never use in the source code itself. I recommend either culling this list down to the essentials for source code work, or renaming this file to `requirements_experiments.txt` and creating a new `requirements.txt` that only contains what you need for the source code to run.

Reply:
I just pushed a commit that updated `requirements.txt` to minimal libraries, while renaming the original file `requirements_experiments.txt`.